tensorboard/plugins/scalar/summary_v2.py

```diff
@@ -30,6 +30,34 @@
 def scalar(name, data, step=None, description=None):
     """Write a scalar summary.
 
+    See also `tf.summary.image`, `tf.summary.histogram`, `tf.summary.SummaryWriter`.
+
+    Logs simple numeric values for later analysis in TensorBoard.  Each summary
```

Comment:
Useful for consistency to mention something here about the current default writer, e.g. like wording suggested in https://github.com/tensorflow/tensorboard/pull/4795/files#r597993135

Would also recommend s/Logs/Writes/ just so we maintain a consistent style throughout.